diagnostics/ENSO_RWS/doc/ENSO_RWS.rst

```diff
@@ -0,0 +1,294 @@
+**Rossby Wave Sources Diagnostic Package**
+
+**Last update 04/15/2021**
+
+ENSO Rossby wave sources (ENSO_RWS) diagnostic package consists of four
+levels. With a focus on identifying leading processes that determine
+ENSO-induced global teleconnection, particularly the Pacific North
+American (PNA) pattern, main module of the POD estimates basic state
```

Comment:
[the] main module of the POD estimates...

Reply:
edited